src/user/screens/starred-repository-list.screen.js

```diff
@@ -52,7 +89,18 @@ class StarredRepositoryList extends Component {
           )}
         {!isPendingStarredRepositories && (
           <FlatList
-            data={starredRepositories}
+            data={starredRepos}
+            onRefresh={() =>
+              this.props.getStarredReposForUser(userId, {
+                forceRefresh: true,
+              })
+            }
+            refreshing={!starredRepos && starredReposPagination.isFetching}
+            onEndReached={() =>
+              this.props.getStarredReposForUser(userId, { loadMore: true })
```

Comment:
`loadMore` tells the proxy to use `nextPageUrl` as endpoint for the next API call, instead of the original endpoint, and the new results are appended to the pagination.